frontend/app/routes/_public+/apply+/$id+/federal-provincial-territorial-benefits.tsx

```diff
@@ -163,13 +164,13 @@ export default function AccessToDentalInsuranceQuestion() {
         )}
       </section>
       <div className="flex flex-wrap items-center gap-3">
-        <ButtonLink id="back-button" to="/apply">
+        <ButtonLink type="button" variant="alternative" to={`/apply/${id}/dental-insurance`}>
```

Comment:
`ButtonLink` doens't have a `type` prop since it's a `link`. Variant should be removed and it with default to `default` variant like other pages.